src/js/background/backgroundLogic.js

```diff
@@ -356,6 +369,11 @@ const backgroundLogic = {
         continue;
       }
 
+      if (tab.groupId && tab.groupId !== TAB_GROUP_ID_NONE) {
```

Comment:
```suggestion
      if (tab.groupId >= 0) {
```

Similar to above. And with this, you can also drop the `TAB_GROUP_ID_NONE` above.